doc/developer/debugging.rst

```diff
@@ -67,20 +67,35 @@ Logs are available in the ``/var/log/postgresql/postgresql-9.*-main.log`` file.
 Makefile
 --------
 
-You can run `DEBUG=TRUE make ...` to have some debug message.
+You can run ``DEBUG=TRUE make ...`` to have some debug message.
```

Comment:
Suggestion:
To obtain additional debug messages, you can rebuild your project as follows: ``DEBUG=TRUE make ...``.